### PR TITLE
RHODS specific changes to main

### PR DESCRIPTION
**Pull request by VaishnaviHire**
This PR updates the main branch with RHODS specific changes

### CODE REVIEW

**Review by etirelli — APPROVED** [review]
lgtm

File: Dockerfiles/bundle.Dockerfile [hunk 1]
```diff
@@ -4,9 +4,9 @@ FROM scratch
 LABEL operators.operatorframework.io.bundle.mediatype.v1=registry+v1
 LABEL operators.operatorframework.io.bundle.manifests.v1=manifests/
 LABEL operators.operatorframework.io.bundle.metadata.v1=metadata/
-LABEL operators.operatorframework.io.bundle.package.v1=opendatahub-operator
-LABEL operators.operatorframework.io.bundle.channels.v1=fast
-LABEL operators.operatorframework.io.bundle.channel.default.v1=fast
+LABEL operators.operatorframework.io.bundle.package.v1=rhods-operator
+LABEL operators.operatorframework.io.bundle.channels.v1=alpha
```
Comment (zdtsw) [inline Dockerfiles/bundle.Dockerfile:8]: do we need to add `embedded` here too?

File: bundle/metadata/annotations.yaml [hunk 2]
```diff
@@ -3,9 +3,9 @@ annotations:
   operators.operatorframework.io.bundle.mediatype.v1: registry+v1
   operators.operatorframework.io.bundle.manifests.v1: manifests/
   operators.operatorframework.io.bundle.metadata.v1: metadata/
-  operators.operatorframework.io.bundle.package.v1: opendatahub-operator
-  operators.operatorframework.io.bundle.channels.v1: fast
-  operators.operatorframework.io.bundle.channel.default.v1: fast
+  operators.operatorframework.io.bundle.package.v1: rhods-operator
+  operators.operatorframework.io.bundle.channels.v1: alpha
```
Comment (zdtsw) [inline bundle/metadata/annotations.yaml:7]: same here

File: config/manifests/kustomization.yaml [hunk 3]
```diff
@@ -14,7 +14,7 @@ patches:
   target:
     group: operators.coreos.com
     kind: ClusterServiceVersion
-    name: opendatahub-operator.v0.0.0
+    name: rhods-operator.v2.0.0
```
Comment (zdtsw) [inline config/manifests/kustomization.yaml:17]: should be v2.1.0 ? same as the one set in Makefile

File: docs/Dev-Preview.md [hunk 4]
```diff
@@ -39,11 +39,11 @@ cat <<EOF | oc create -f -
 apiVersion: operators.coreos.com/v1alpha1
 kind: Subscription
 metadata:
-  name: opendatahub-operator
+  name: rhods-operator
```
Comment (zdtsw) [inline docs/Dev-Preview.md:42]: should line 25 be updated too?

File: Dockerfiles/bundle.Dockerfile (hunk 1 above)
Comment (VaishnaviHire) [reply Dockerfiles/bundle.Dockerfile]: Updated

File: bundle/metadata/annotations.yaml (hunk 2 above)
Comment (VaishnaviHire) [reply bundle/metadata/annotations.yaml]: Updated

File: config/manifests/kustomization.yaml (hunk 3 above)
Comment (VaishnaviHire) [reply config/manifests/kustomization.yaml]: Updated

**Comment by VaishnaviHire** [comment]
@zdtsw Addressed all comments. Can you test this before merging ?

File: docs/Dev-Preview.md (hunk 4 above)
Comment (VaishnaviHire) [reply docs/Dev-Preview.md:42]: Dev preview is only applicable for ODH, so I think we should revert all changes in this doc

**Comment by zdtsw** [comment]
> @zdtsw Addressed all comments. Can you test this before merging ?

sure, i will do a local build and test it first

**Comment by zdtsw** [comment]
local build from `VaishnaviHire:rhods_changes` : quay.io/wenzhou/opendatahub-operator-catalog:v2.1.47

**Comment by zdtsw** [comment]
test on the local build: 
codeflare + ray => checked
dashbaord => fix  in https://github.com/red-hat-data-services/odh-manifests/pull/485
kserve => checked
dsp => keep reconcile, 2 RS using different images???? 
modelserving => cannot enabled after kserve was previous enabled, known issue

**Comment by zdtsw** [comment]
> test on the local build: codeflare + ray => checked dashbaord => fix in [red-hat-data-services/odh-manifests#485](https://github.com/red-hat-data-services/odh-manifests/pull/485) kserve => checked dsp => keep reconcile, 2 RS using different images???? modelserving => cannot enabled after kserve was previous enabled, known issue

so what i observe now is:
DSP created 2 Configmap, one is using DSPO image from registry.redhat.io and another one is using the one from quay.io (which is the value set in ODH odh-manifests)
i cannot see how it gets this value and i do not have ODH instance running in the cluster, so it is very confusing for this case. 

it would be good if anyone can either use my local build image, or do a clean build with same code base and test DSP part.

File: config/manifests/bases/rhods-operator.clusterserviceversion.yaml [hunk 5]
```diff
@@ -0,0 +1,127 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: '[]'
+    capabilities: Full Lifecycle
```
[the remark this answers is not in the document]
Comment (VaishnaviHire) [reply config/manifests/bases/rhods-operator.clusterserviceversion.yaml:6]: I think it is okay to have this Full Lifecycle

File: controllers/dscinitialization/dscinitialization_controller.go [hunk 6]
```diff
@@ -244,10 +244,7 @@ var singletonPredicate = predicate.Funcs{
 
 	CreateFunc: func(e event.CreateEvent) bool {
 		if e.Object.GetObjectKind().GroupVersionKind().Kind == "DSCInitialization" {
-			if e.Object.GetName() == "default" {
-				return true
-			} else {
-				// Set to error level since it causes Panic
+			if e.Object.GetName() != "default" {
```
[the remark this answers is not in the document]
Comment (VaishnaviHire) [reply controllers/dscinitialization/dscinitialization_controller.go:247]: I will remove this change from the PR, will open a PR in upstream

**Comment by VaishnaviHire** [comment]
Can you share your build ?

**Comment by zdtsw** [comment]
> quay.io/wenzhou/opendatahub-operator-catalog:v2.1.47

i used this one  quay.io/wenzhou/opendatahub-operator-catalog:v2.1.47

**Comment by VaishnaviHire** [comment]
@zdtsw Update with latest dashboard changes

**Pull request merged**